integration/use-cases/02-create-private-registry.js

```diff
@@ -2,7 +2,11 @@ const axios = require("axios");
 const utils = require("./lib/utils");
 
 test("Creates a private registry", async () => {
-  var token = process.env.USE_MULTICLUSTER_OIDC_ENV ? undefined : process.env.ADMIN_TOKEN;
+  var token =
+    process.env.USE_MULTICLUSTER_OIDC_ENV === true ||
+      process.env.USE_MULTICLUSTER_OIDC_ENV === "true"
```

Comment:
an env var is always going to be a string

Reply:
Ooops, should have checked it, sorry. Changing it in the #2651